packages/x-charts/src/RadarChart/seriesConfig/seriesProcessor.ts

```diff
@@ -0,0 +1,29 @@
+import { type SeriesProcessor } from '../../internals/plugins/models/seriesConfig';
+import type { DefaultizedRadarSeriesType } from '../../models';
+import type { SeriesId } from '../../models/seriesType/common';
+
+const radarValueFormatter = ((v) =>
+  v == null ? '' : v.toLocaleString()) as DefaultizedRadarSeriesType['valueFormatter'];
```

Comment:
Do we need the cast here? 

Can't we do this?

```suggestion
const radarValueFormatter: DefaultizedRadarSeriesType['valueFormatter'] = ((v) =>
  v == null ? '' : v.toLocaleString());
```